deploy/crds/ocs.openshift.io_storageclusters_crd.yaml

```diff
@@ -181,11 +181,7 @@ spec:
                             resources allowed. More info: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/'
                           type: object
                           additionalProperties:
-                            pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
-                            anyOf:
-                            - type: integer
-                            - type: string
-                            x-kubernetes-int-or-string: true
+                            type: string
```

Comment:
Its strange that `verify-latest-csv` is failing until and unless I won't commit these changes that gets generated upon running `make-update-generated` on my local machine!.